src/main/java/me/totalfreedom/totalfreedommod/TotalFreedomMod.java

```diff
@@ -296,83 +325,100 @@ public Initializer()
-            wr = new WorldRestrictions();
-            pl = new PlayerList();
-            sh = new Shop();
-            vo = new Votifier();
-            an = new Announcer();
-            cm = new ChatManager();
-            dc = new Discord();
-            pul = new PunishmentList();
-            bm = new BanManager();
-            im = new IndefiniteBanList();
-            pem = new PermissionManager();
-            gr = new GameRuleHandler();
-            snp = new SignBlocker();
-            ew = new EntityWiper();
-            st = new Sitter();
-            vh = new VanishHandler();
-            ptero = new Pterodactyl();
+            adminList = new AdminList();
+            activityLog = new ActivityLog();
+            rankManager = new RankManager();
+            commandBlocker = new CommandBlocker();
+            eventBlocker = new EventBlocker();
+            blockBlocker = new BlockBlocker();
+            mobBlocker = new MobBlocker();
+            interactBlocker = new InteractBlocker();
+            potionBlocker = new PotionBlocker();
+            loginProcess = new LoginProcess();
+            antiNuke = new AntiNuke();
+            antiSpam = new AntiSpam();
+            playerList = new PlayerList();
+            shop = new Shop();
+            votifier = new Votifier();
+            announcer = new Announcer();
+            chatManager = new ChatHandler();
+            discord = new Discord();
+            punishmentList = new PunishmentList();
+            banManager = new BanManager();
+            indefiniteBanList = new IndefiniteBanList();
+            permissionManager = new PermissionManager();
+            signBlocker = new SignBlocker();
+            entityWiper = new EntityWiper();
+            sitter = new Sitter();
+            vanishHandler = new VanishHandler();
+            pterodactyl = new Pterodactyl();
+
+            if (isPluginPresent("WorldGuard"))
+                worldRestrictions = new WorldRestrictions();
         }
 
         private void initAdminUtils()
         {
             // Single admin utils
-            cs = new CommandSpy();
-            ca = new Cager();
-            fm = new Freezer();
-            or = new Orbiter();
-            mu = new Muter();
-            ebl = new EditBlocker();
-            pbl = new PVPBlocker();
-            fo = new Fuckoff();
-            ak = new AutoKick();
-            ae = new AutoEject();
-            mo = new Monitors();
+            commandSpy = new CommandSpy();
+            cager = new Cager();
+            freezer = new Freezer();
+            orbiter = new Orbiter();
+            muter = new Muter();
+            editBlocker = new EditBlocker();
+            pvpBlocker = new PVPBlocker();
+            fuckoff = new Fuckoff();
+            autoKick = new AutoKick();
+            autoEject = new AutoEject();
+            monitors = new Monitors();
         }
 
         private void initBridges()
         {
             // Start bridges
```

Comment:
We don't seem to tell the server to load these plugins prior to TFM, we should be doing it if we're going to intentionally not load up aspects of TFM based on them running / not.